pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/Statistics.sql.stg

```diff
@@ -0,0 +1,87 @@
+group Statistics;
+
+getParticipantStatistics() ::= <<
+select
+    case when ug.user_governance_id is null then 'self' else 'governed' end as user_type,
+    count(1) as user_count
+from
+    user_study_enrollment se
```

Comment:
When querying from the `user_study_enrollment` table, we should narrow down the result set by adding `se.valid_to is null` to only get rows with the latest status. Reason is that this table contains historical status as well (hence the `valid_from` and `valid_to` columns).

Reply:
Fixed